app/src/app/present-round/present-round.component.ts

```diff
@@ -45,9 +48,69 @@ export class PresentRoundComponent implements OnInit {
         return val as boolean
       })
     )
+    this.markedDecreeAsSent = this.db.object("delegateRounds/" + this.delegateId + "/" + this.roundId + "/markedDecreeAsSent").valueChanges().pipe(
+      map(val => {
+        return val as boolean
+      })
+    )
+    this.db.list("landsraad/questions").snapshotChanges().pipe(
+      tap(
+        questionsSnapshots => {
+          this.availableQuestions = questionsSnapshots.map(qSnapshot => {
+            const question: any = qSnapshot.payload.val()
+            const qKey = qSnapshot.key
+            return {
+              ...question,
+              id: qKey
+            }
+          })
+        }
+      )
+    ).subscribe(ref => this.controlledVotingRights = this.db.list("landsraad/votingRights", ref => ref.orderByChild("controlledBy").equalTo(this.delegateId)).snapshotChanges().pipe(
```

Comment:
Don't subscribe here. Subscriptions is done when you do `| async` in the template. Just use map/flatmaps to get data. Get inspired by how actions are done.

Reply:
This also got solved by refactor.